pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -295,16 +297,19 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 		if err := bootstrapSCCScheme.Convert(v1constraint, constraint, nil); err != nil {
 			panic(err)
 		}
-		constraints[i] = constraint
 
 		if usersToAdd, ok := sccNameToAdditionalUsers[constraint.Name]; ok {
-			constraints[i].Users = append(constraints[i].Users, usersToAdd...)
+			constraint.Users = append(constraint.Users, usersToAdd...)
 		}
 		if groupsToAdd, ok := sccNameToAdditionalGroups[constraint.Name]; ok {
-			constraints[i].Groups = append(constraints[i].Groups, groupsToAdd...)
+			constraint.Groups = append(constraint.Groups, groupsToAdd...)
 		}
+		if err := securityapisv1.Convert_security_SecurityContextConstraints_To_v1_SecurityContextConstraints(constraint, v1constraint, nil); err != nil {
```

Comment:
You should create the SCCs as v1 right away, no nasty conversions at the end, probably just the defaulting. That does not explain the test failures though.

Reply:
I tried that (will try again maybe i'm remembering wrong) but there are diffs w/ external,internal types, couldn't make it work-lemme look/try again

Reply:
ok fixed